### PR TITLE
Added Timestamp precision in Trino 

### PR DESCRIPTION
**Pull request by Luishfs**
# Description

Allows for Trino SQL and Python Blocks to insert TIMESTAMP precision to the trino loader
This allows for Trino to be used with S3 Iceberg, which default TIMESTAMP value is 6

# How Has This Been Tested?

Tested on a local Trino and Mage instance

Default TIMESTAMP value:
Screenshot from 2023-09-10 23-43-17

Modified TIMESTAMP value using precision:
Screenshot from 2023-09-10 23-43-48

io_config example:
Screenshot from 2023-09-11 01-25-52

# Checklist
- [X] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

cc:
@wangxiaoyou1993

### CODE REVIEW

File: docs/guides/io_config.mdx [hunk 1]
```diff
@@ -116,4 +116,5 @@ default:
     source: trino-cli                         # Optional
     user: admin
     verify: /path/to/your/ca.crt              # Optional
+    precision: 3                              # Optional. Set TIMESTAMP(n) precision
```
Comment (wangxiaoyou1993) [inline docs/guides/io_config.mdx:119]: let's put it under `data_type_propereties` like:
```
data_type_propereties
  timestamp_precision
``

Comment (Luishfs) [reply docs/guides/io_config.mdx]: Modified on both .mdx and code in 3e4566b7fb71b918ec79c0fff3164eaed6e8f545

File: mage_ai/shared/utils.py [hunk 2]
```diff
@@ -93,15 +96,17 @@ def convert_python_type_to_bigquery_type(python_type):
...
         return 'BIGINT'
     elif python_type is float:
         return 'DOUBLE'
     elif python_type is bool:
         return 'BOOLEAN'
-    elif python_type is datetime:
+    elif python_type is datetime and usr_data_types is None:
         return 'TIMESTAMP'
+    elif python_type is datetime and usr_data_types is not None:
+        return f'TIMESTAMP({usr_data_types.get("timestamp_precision")})'
```
Comment (wangxiaoyou1993) [inline mage_ai/shared/utils.py:109]: if `usr_data_types` doesn't have `timestamp_precision` key, it'll throw error
also, we prefer using single quote in the code

Comment (Luishfs) [reply mage_ai/shared/utils.py]: Modified in fd632852c12c167b2638456d98769f07b19c0d5a

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**